src/biblebot/auth.py

```
@@ -0,0 +1,250 @@
+            for child in store.glob("**/*"):
+                try:
+                    child.unlink()
+                except IsADirectoryError:
+                    pass
+            # remove empty dirs
+            for root, dirs, files in os.walk(store, topdown=False):
+                for name in files:
+                    try:
+                        (Path(root) / name).unlink()
+                    except Exception:
+                        pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)